### DISCUSSION TITLE
Securly v130

### DISCUSSION THREAD

**Discussion by Sam-hacks1** [3kh0/ext-remover#1270] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
what exploits can I use for securly v130?

**Comment by akabutnicer** [comment]
your tears

**Comment by akabutnicer** [comment]
anyways I'm working on a skiovox for current chrome versions

**Reply by Balistic123** [reply]
need any help?

**Reply by akabutnicer** [reply]
no...i don't 🙄

**Comment by Agguiopz** [comment]
**The Power of Friendship**

**Comment by S-PScripts** [comment]
dupe https://github.com/3kh0/ext-remover/discussions/1314
